visualization/app/codeCharta/ui/fileChooser/fileChooser.component.ts

```diff
@@ -16,6 +17,7 @@ export class FileChooserController {
 	constructor(private $scope, private codeChartaService: CodeChartaService, private storeService: StoreService) {}
 
 	onImportNewFiles(element) {
+		UnfocusButtonController.NEW_FILES_LOADED = true
```

Comment:
Wasn't this possible with listening to en event?

Reply:
I actually don't know)

Reply:
Well, it was pretty simple and it's done now :)